### DISCUSSION TITLE
Can mapperly map a sealed record?

### DISCUSSION THREAD

**Discussion by poeter** [riok/mapperly#270] [Q&A]
Trying to map a sealed record to a class but I'm not seeing the mapping generated. Is this use case supported?

Thanks

**Comment by latonz** [comment]
Yes this use case should be supported. Could you provide more details (Mapperly version used, Target Framework, Class with the [Mapperly] attribute, other involved classes, ...)?

**Reply by poeter** [reply]
Thanks Iatonz. All good. Somehow this resolved itself when I re-installed my visual studio.
